pyuvsim/simsetup.py

```diff
@@ -1012,7 +1017,7 @@ def initialize_uvdata_from_params(obs_params):
     uv_obj.history = ''
 
     # select on object
-    valid_select_keys = ['antenna_nums', 'antenna_names', 'ant_str', 'bls', 'frequencies', 'freq_chans', 'times', 'polarizations', 'blt_inds']
+    valid_select_keys = ['antenna_nums', 'antenna_names', 'ant_str', 'bls', 'frequencies', 'freq_chans', 'times', 'blt_inds']
```

Comment:
why did polarizations get taken out as a valid selection key?

Reply:
It seemed like a strange option to have, since we're defining a UVData object and not selecting on an existing one. I don't see any reason we can't keep it, I'm just not sure what the use case is.